infrastructure/lib/atat-web-api-stack.ts

```diff
@@ -264,7 +337,7 @@ export class AtatWebApiStack extends cdk.Stack {
         // portfolios to never be processed.
         batchSize: 1,
         method: HttpMethod.GET,
-        handlerPath: this.determineApiHandlerPath("consumePortfolioDraftSubmitQueue", "portfolioDrafts/submit/"),
+        handlerPath: this.determineApiHandlerPath("consumePortfolioDraftSubmitQueue", "provision/"),
```

Comment:
@jeffsegal-ccpo  The `consumePortfolioDraftSubmitQueue` is a lambda polling from the SQS Submit Queue, so this would not be directly called by the API. Are you referring to the `submitPortfolioDraft` lambda which currently places a message on the SQS Submit Queue to provision the submitted resources? I think the `submitPortfolioDraft` aligns with or will be replaced by the `/portfolios/:id/submit` if my understanding is correct.

Reply:
Ahh got it, yeah sorry I just sped through too fast and made an incorrect assumption.